src/components/NavBar/NavMenu.jsx

```diff
@@ -52,32 +52,27 @@ const NavMenu = ({ menuId, openMenu, setOpenMenu, anchorEl, setAnchorEl, setShow
             Notifications
```

Comment:
Just noticed this have shrinking the screen for the app.

https://github.com/codeforpdx/PASS/assets/14917816/f8a76544-9610-4d1c-8f0b-ef2cd520cb64

Believe we could do the same modifications we've done for "Profile", "Setting", and "Log out" in this PR and extend it to Notifications.

Looks like we just need to change the base component for the first `<MenuItem>` into a button via `component={Button}`

Reply:
Oh wow - I hadnt noticed the Notifications component jumping into the menu upon screen resize! I'll take a look at that this evening to see if I can match the appearance to the other MenuItems.

Reply:
@leekahung @cameronabel 

that was my doing.  We discussed it way back in PR #141  - Check the video there.

I'd have to look at Figma file if we have something different now, but basically on mobile screens all items move into a hamburger menu as is convention. 

I believe the only reason it looks weird right now is because we only have 2 icons on right side of navbar, but honestly the "Messages" route should be a mail icon over by notifications bell icon as well.  So all 3 would go into the hamburger menu on small screens.

I'm still not sure we need the dropdown menu for the routes (Clients/Documents) - I feel that can just always stay as tabs.  But again, depends on what we add in the future.

Reply:
For me, I'm just pointing out the styling for "Notifications". Think could deal with other issues related to this in another PR.

Reply:
### Oh gotcha! I misunderstood...

Reply:
A little tangential but I've noticed that, when opened, Menu and Notifications use a light green backdrop. This is unlike the `AddClientModal.jsx` and `LogoutModal.jsx` (which have a gray backdrop) and `NewMessage.jsx` (which has a custom blur effect).

`NewMessage.jsx` will be refactored with MUI, so that ought to resolve itself. But in general, I think we should decide on a set style for all modals, pop-ups, etc. Such as whether to stick with gray, green, or something else.

Personally, I think the gray backdrop is a bit nicer as well as more standard-looking. The green is a tad distracting in my opinion, even though it does fit in with our current color scheme (except we may not even end up sticking with that). Also, the gray is what comes default with MUI modals.

Reply:
> A little tangential but I've noticed that, when opened, Menu and Notifications use a light green backdrop. This is unlike the `AddClientModal.jsx` and `LogoutModal.jsx` (which have a gray backdrop) and `NewMessage.jsx` (which has a custom blur effect).
> 
> `NewMessage.jsx` will be refactored with MUI, so that ought to resolve itself. But in general, I think we should decide on a set style for all modals, pop-ups, etc. Such as whether to stick with gray, green, or something else.
> 
> Personally, I think the gray backdrop is a bit nicer as well as more standard-looking. The green is a tad distracting in my opinion, even though it does fit in with our current color scheme (except we may not even end up sticking with that). Also, the gray is what comes default with MUI modals.

Yeah, something I did keeping it all "on theme".  But agree, kind of looks a little tacky.   

And yeah we'd have to deep dive into the `theme.js` if we wanted it all to be the green as styling the modal backdrops with sx didn't seem to work no matter what I tried.

But either direction, again consistency. All grey or all green.  But seeing them both, I think the grey looks better honestly.

Reply:
+1 for having the gray backdrop for modals.
-1 for Scott spelling gray like a british person. This Is America.

Reply:
> +1 for having the gray backdrop for modals.
> -1 for Scott spelling gray like a british person. This Is America.

“Gray” is a last name, “grey” is a color - despite the last decade and a half of Americans changing it because they don’t know the difference 😂